src/main/java/supertracker/parser/Parser.java

```diff
@@ -139,7 +139,7 @@ public class Parser {
             "(?<" + FROM_GROUP + ">(?:" + FROM_FLAG + BASE_FLAG + ".*)?) " +
             "(?<" + TO_GROUP + ">(?:" + TO_FLAG + BASE_FLAG + ".*)?) ";
     private static final String RENAME_COMMAND_REGEX = NAME_FLAG + BASE_FLAG + "(?<" + NAME_GROUP + ">.*) " +
-            "(?<" + NEW_NAME_GROUP + ">(?:" + NEW_NAME_FLAG + BASE_FLAG + ".*)?) ";
+            NEW_NAME_FLAG + BASE_FLAG + "(?<" + NEW_NAME_GROUP + ">.*) ";
```

Comment:
Good change on the regex so that it is consistent for NEW_NAME_FLAG to be a compulsory flag